server/googleapi/googleCoordinates.js

```diff
@@ -29,7 +31,18 @@ async function getCoordinatesFromLocation(
       },
     })
     if (!response.data.results.length) {
-      throw new Error('No coordinates found.')
+      console.log(destinationCity, stageLocationName)
```

Comment:
another console log that maybe should remove? :3 up to u tho

Reply:
this one is intentional